src/widgets/index/index.ts

```diff
@@ -348,11 +348,15 @@ const index = (props: IndexProps): Index => {
         if (__DEV__) {
           // Some connectors are responsible for multiple widgets so we need
           // to map them.
+          // eslint-disable-next-line no-inner-declarations
```

Comment:
why do we even have this eslint rule? seems useless to me

Reply:
It's part of `eslint:recommended`. I agree that's not a great rule, especially here since we create that function for each index (i.e. not often).